mongodb/curd.go

```diff
@@ -0,0 +1,718 @@
+package mongodb
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// MongoArgFilter mongo operation instruction
+const (
+	MongoArgFilter               = "filter"
```

Comment:
These constants do not need to be exported

Reply:
modify these constants to be non-exported